openqabot/pc_helper.py

```diff
@@ -62,12 +59,7 @@ def apply_publiccloud_regex(settings):
             settings["PUBLIC_CLOUD_IMAGE_REGEX"]
         )
         return settings
-    except (
-        ConnectionError,
-        Timeout,
-        ValueError,
-        re.error,
-    ) as e:
+    except (ConnectionError, Timeout, ValueError, re.error, HTTPError) as e:
```

Comment:
Is the necessary? The `get_latest_pc_image()` calls `fetch_matching_link()` where we are already checking for those exceptions.

Reply:
IDK .. its code written by PubCloud team :D and really bigger changes aren't target of this PR :D